crates/primitives/src/sorted_vec.rs

```diff
@@ -0,0 +1,194 @@
+use std::cmp::Ordering;
```

Comment:
Have you checked?
https://crates.io/crates/sorted-vec or https://crates.io/crates/sortedvec ?

Reply:
Yes, I had looked into the first one. The second one seems to use macro magic. I just wanted to keep the dependencies minimal for small added feature.

Reply:
Ok makes sense. Thanks!